.circleci/config.yml

```diff
@@ -54,5 +45,4 @@ workflows:
   tests:
     jobs:
       - image-tests-mpl302
-#      - image-tests-mpl310
```

Comment:
This was just a temporary disabling, I suppose we still want to check the latest MPL release, not just the oldest supported and dev

Reply:
Yes we used to actually check all supported versions so we should probably reinstate that?

Reply:
Okay, after I figure out the mess with the other two jobs, I'll add this back.

Do you remember why we disabled it?

Reply:
yep, this was failing in the np version drop PR as it needed the image update, so it all referred back to the python drop PR to get fixed. See https://github.com/astropy/astropy/pull/11935#issuecomment-877692730

Reply:
OK, I re-enabled it. The `mpl310` job name was misleading because it really was calling `mpl311`, so I updated that as well.